src/main/java/com/github/rodionovsasha/commentservice/entities/User.java

```diff
@@ -28,6 +29,10 @@
     @Setter
     private boolean active = true;
 
+    @JsonIgnore
+    @OneToMany(mappedBy = "owner", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Topic> topics = new LinkedList<>();
```

Comment:
Why linked list?

Reply:
Also, @rodionovsasha please ensure that topics are lazy loaded

Reply:
OneToMany FetchType fetch() default LAZY;